src/Spatial/Euclidean/Point2D.cs

```diff
@@ -361,8 +361,8 @@ void IXmlSerializable.ReadXml(XmlReader reader)
         /// <inheritdoc />
         void IXmlSerializable.WriteXml(XmlWriter writer)
         {
-            writer.WriteAttribute("X", this.X);
-            writer.WriteAttribute("Y", this.Y);
+            writer.WriteElement("X", this.X, "G15");
```

Comment:
Changing from attribute to element is a breaking change. Elements would probably have been the better choice.

`"R"` is a better format specifier. https://docs.microsoft.com/en-us/dotnet/standard/base-types/standard-numeric-format-strings#RFormatString